### ISSUE TITLE
Is the primary authentication check logic missing?

### ISSUE THREAD

**Issue by vresetnikov** [KenAli77/InFit#3]
Hello,

Looking at the `master` branch it doesn't seem like the app performs any checks whether the user is logged in on startup? 

Looks like the app starts with the "Login" screen as the initial screen. 
The login can then be performed. 

However, I could not find in the code anything that handles the subsequent app launches and verifies that the user is logged in?

**Comment by KenAli77** [comment]
You're right. The feature is not fully complete. I made a branch you can work on if you feel like it. [3-primary-authentication-check]
